README.md

```diff
@@ -101,20 +104,20 @@ The format for a structured profile is below:
             "unique_count": int,
             "unique_ratio": float,
             "precision": {
-	        'min': int,
-		'max': int,
-		'mean': float,
-		'var': float,
-		'std': float,
-		'sample_size': int,
-		'margin_of_error': float,
-		'confidence_level': float		
-	    },
+                'min': int,
+                'max': int,
+                'mean': float,
+                'var': float,
+                'std': float,
+                'sample_size': int,
+                'margin_of_error': float,
+                'confidence_level': float		
+            },
```

Comment:
The formatting when viewing the md file was not readable in terms of indenting, I made it match the rendered README